_posts/2016-04-14-GitHubPages.markdown

```diff
@@ -0,0 +1,9 @@
+---
+title: Web design with Jekyll/Github Pages
```

Comment:
A small change: it's GitHub (there are several other capitalizations used below.)